sidekiq_alive.gemspec

```diff
@@ -50,7 +50,6 @@ Gem::Specification.new do |spec|
   spec.add_development_dependency("rubocop-shopify", "~> 2.10")
   spec.add_development_dependency("solargraph", "~> 0.49.0")
 
-  spec.add_dependency("rack", "< 3")
+  spec.add_dependency("gserver", "~> 0.0.1")
```

Comment:
gserver used to be part of standard lib so it's fairly light weight and doesn't have any transitive dependencies